Jenkinsfile

```
@@ -40,5 +40,6 @@
 // I.e. for testing library changes
 //@Library(value="pipeline-lib@your_branch") _
 
-packageBuildingPipeline(['distros' : ['el8', 'leap15']])
+packageBuildingPipeline(['distros' : ['el8', 'leap15'],
```

You need to implement the [suggestion](https://github.com/daos-stack/mpifileutils-pkg/pull/27/files/6588fb5ea33f99fdda4a09269e0e4cbbf3595978#r951878070) completely:

```suggestion
packageBuildingPipelineDAOSTest(['distros' : ['el8', 'leap15'],
```

---

Thank you very much! Just updated it.